src/components/sample/sample.jsx

```diff
@@ -0,0 +1,9 @@
+import React, { Component } from 'react';
+
+export default class Sample extends Component {
+    render() {
+        return (
```

Comment:
Basically, it's 2 spaces. We set up a pre-commit linting hook for Airbnb style guide. If any of the linting rules fails then it won't commit. I guess this indentation is two spaces. I would check